### ISSUE TITLE
SUL logo links should have discernible text

### ISSUE THREAD

**Issue by alundgard** [sul-dlss/component-library#74]
### Design
The Stanford University Libraries logo `<a>` elements should have discernible text.

`<a ... >Stanford University Libraries</a>`

Note: We are now using the full name "Stanford University Libraries" — not "Stanford Libraries".

### Current
**Header** code reference
`<a class="mb-0 navbar-brand navbar-logo" href="https://library.stanford.edu">Stanford Libraries</a>`

**Footer** code reference
`<a href="https://library.stanford.edu" class="prefooter-logo"></a>`
